### ISSUE TITLE
Improving Multi-fidelity Optimization with a Recurring Learning Rate for Hyperparameter Tuning

### ISSUE THREAD

**Issue by nabenabe0928** [nabenabe0928/reading-list#72]
[Improving Multi-fidelity Optimization with a Recurring Learning Rate for Hyperparameter Tuning](https://arxiv.org/abs/2209.12499)

SHA + learning rate scheduling (I do not know why LR scheduler is out of HPO, but anyways, they propose like this)

## Main points
- point 1
- point 2

## Confusing points

1. Section 1
- point 1
- point 2
